examples/animation/animated_fox.rs

```diff
@@ -0,0 +1,127 @@
+}
+
+struct Animations(Vec<Handle<AnimationClip>>);
+
+fn setup(
+    mut commands: Commands,
+    asset_server: Res<AssetServer>,
+    mut scene_spawner: ResMut<SceneSpawner>,
+    mut meshes: ResMut<Assets<Mesh>>,
+    mut materials: ResMut<Assets<StandardMaterial>>,
+) {
+    // Insert a resource with the current scene information
+    commands.insert_resource(Animations(vec![
+        asset_server.load("models/animated/Fox.glb#Animation2"),
+        asset_server.load("models/animated/Fox.glb#Animation1"),
+        asset_server.load("models/animated/Fox.glb#Animation0"),
+    ]));
+
+    // Camera
+    commands.spawn_bundle(PerspectiveCameraBundle {
+        transform: Transform::from_xyz(100.0, 100.0, 150.0)
+            .looking_at(Vec3::new(0.0, 20.0, 0.0), Vec3::Y),
+        ..Default::default()
+    });
+
+    // Plane
+    commands.spawn_bundle(PbrBundle {
+        mesh: meshes.add(Mesh::from(shape::Plane { size: 500000.0 })),
+        material: materials.add(Color::rgb(0.3, 0.5, 0.3).into()),
+        ..default()
+    });
+
+    // Light
+    commands.spawn_bundle(DirectionalLightBundle {
+        transform: Transform::from_rotation(Quat::from_euler(
+            EulerRot::ZYX,
+            0.0,
+            1.0,
+            -std::f32::consts::FRAC_PI_4,
+        )),
+        directional_light: DirectionalLight {
+            shadows_enabled: true,
+            ..default()
+        },
+        ..default()
+    });
+
+    // Fox
+    scene_spawner.spawn(asset_server.load("models/animated/Fox.glb#Scene0"));
+
+    println!("Animation controls:");
+    println!("  - spacebar: play / pause");
+    println!("  - arrow up / down: speed up / slow down animation playback");
+    println!("  - arrow left / right: seek backward / forward");
+    println!("  - return: change animation");
+}
+
+// Once the scene is loaded, start the animation
+fn setup_scene_once_loaded(
+    animations: Res<Animations>,
+    mut player: Query<&mut AnimationPlayer>,
+    mut done: Local<bool>,
+) {
+    if !*done {
+        if let Ok(mut player) = player.get_single_mut() {
+            player.play_animation(animations.0[0].clone_weak()).repeat();
+            *done = true;
+        }
+    }
+}
+
+fn keyboard_animation_control(
+    keyboard_input: Res<Input<KeyCode>>,
+    mut animation_player: Query<&mut AnimationPlayer>,
+    animations: Res<Animations>,
+    mut current_animation: Local<usize>,
+) {
+    if let Ok(mut player) = animation_player.get_single_mut() {
+        if keyboard_input.just_pressed(KeyCode::Space) {
+            if player.is_paused() {
+                player.play();
+            } else {
+                player.pause();
+            }
+        }
+
+        if keyboard_input.just_pressed(KeyCode::Up) {
+            let speed = player.speed();
+            player.set_speed(speed * 1.2);
+        }
+
+        if keyboard_input.just_pressed(KeyCode::Down) {
+            let speed = player.speed();
+            player.set_speed(speed * 0.8);
```

Comment:
```suggestion
            player.set_speed(speed * 0.833333333);
```
Multiplying by 1.2 and then by 0.8 does not get you back to where you started.
Not a big deal though. I like that you're using multiplication.

Reply:
Hmm. but multiplication can't make speed negative to play the animation in reverse.

Reply:
I would prefer to keep 0.8 for simplicity sake. Same for putting a negative speed